binaries/org.eclipse.swt.win32.win32.aarch64/build.properties

```diff
@@ -0,0 +1,40 @@
+###############################################################################
+# Copyright (c) 2023, 2024 Tue Ton and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     Tue Ton - initial API and implementation
+#     Hannes Wellmann - Leverage Tycho pomless
+###############################################################################
+custom = true
+bin.includes = .,*.dll,about_files/,about.html,fragment.properties
+bin.excludes = library/
+source.. = \
+	../../bundles/org.eclipse.swt/Eclipse SWT/win32,\
+	../../bundles/org.eclipse.swt/Eclipse SWT/common,\
+	../../bundles/org.eclipse.swt/Eclipse SWT PI/common,\
+	../../bundles/org.eclipse.swt/Eclipse SWT PI/win32,\
+	../../bundles/org.eclipse.swt/Eclipse SWT OLE Win32/win32,\
+	../../bundles/org.eclipse.swt/Eclipse SWT Accessibility/win32,\
+	../../bundles/org.eclipse.swt/Eclipse SWT Accessibility/common,\
+	../../bundles/org.eclipse.swt/Eclipse SWT AWT/win32,\
+	../../bundles/org.eclipse.swt/Eclipse SWT AWT/common,\
+	../../bundles/org.eclipse.swt/Eclipse SWT Drag and Drop/win32,\
+	../../bundles/org.eclipse.swt/Eclipse SWT Drag and Drop/common,\
+	../../bundles/org.eclipse.swt/Eclipse SWT Printing/win32,\
+	../../bundles/org.eclipse.swt/Eclipse SWT Printing/common,\
+	../../bundles/org.eclipse.swt/Eclipse SWT Program/win32,\
+	../../bundles/org.eclipse.swt/Eclipse SWT Program/common,\
+	../../bundles/org.eclipse.swt/Eclipse SWT Custom Widgets/common,\
+	../../bundles/org.eclipse.swt/Eclipse SWT Browser/common,\
+	../../bundles/org.eclipse.swt/Eclipse SWT Browser/win32,\
+	../../bundles/org.eclipse.swt/Eclipse SWT OpenGL/win32,\
+	../../bundles/org.eclipse.swt/Eclipse SWT OpenGL/common
+src.includes = about.html,about_files/
```

Comment:
These files are missing as well.

Reply:
The `build.properties` and `MANIFEST.MF` files are not really needed to produce the SWT natives for this PR; they are needed so that the Maven build script would work with the following command to generate the SWT natives:
```
mvn clean process-resources -Dnative=win32.win32.aarch64
```
because the build script requires them to be there, hence they are included in this PR.

The above files are really part of my second PR that will actually produce the `win32.aarch64` module's jar and zip files which will include these SWT natives. I was waiting for this PR to be merged, so that the SWT natives would be produced and checked in, before submitting the second PR. But now you've mentioned these files, and to avoid confusion, I will submit the second PR quickly, to complete my work for Windows on Arm64 support for this SWT repo.

Reply:
I don't think it makes sense to have both PRs split... can we not just use one PR and maybe just comment out the module if it should not been build yet by maven?

Reply:
#1083 submitted, to complement this PR.

Reply:
The separate PRs serve distinct aims: this PR is for producing the `SWT natives` for Windows Arm64, while the other one (#1083) is for building the SWT jar/zip files. The second one is dependent on this PR to be merged first and be generated the official `swt*.dll` files, because the second PR would fail to build if the `*.dll` files are missing.

But if you insist, I can merge the two PRs into this one, but will leave the SWT jar/zip files not built, and someone will have to modify it later to get it built by Maven, once the `swt*.dll` are generated and checked in to the repo.

Reply:
I  would say lets wait for hannes, but the checkked in binaries afaik are only for developer convenience now and are rebuild anyways

Reply:
Yes please be so kind and combine both changes in PR. Otherwise we can't verify that everything is working.
And without the native build Jenkins agents we can't submit any binary as they would get deleted or out of sync.
If you prefer you can still have both changes in separated commits in this PR.

Btw. there have been changes in the configuration of the fragments recently that you probably should pick up.

Reply:
I've merged PR #1083 to this PR, rebased to the latest master branch and updated this branch according.